modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/DDMFormValuesFactory.java

```diff
@@ -31,7 +31,7 @@ public DDMFormValues create(
 
 	public DDMFormValues create(PortletRequest portletRequest, DDMForm ddmForm);
 
-	public boolean isAnyDDMFormValueEdited(
+	public boolean hasDDMFormValueEdited(
```

Comment:
Hey @carolmariaabb , there is a lot of logic to be moved to JournalEditArticleDisplayContext, maybe they ask me do the contrary. Did you discuss it with some member of team Echo? 
Since the logic is quite similar to the current one of the factory, I decided to place it there.

Reply:
@crodriguezy what do you think about creating a `Util` class so you can reuse some code? We need to avoid creating this method in this factory.

Reply:
Hi @carolmariaabb , the Util class may be a better option, yes, but what about adding a method called `createFromRequest` in this factory? The same logic of `create` would apply if no value has been edited.

Reply:
> there is a lot of logic to be moved to JournalEditArticleDisplayContext

@crodriguezy can you show me which methods would need to be duplicated in this case?

Reply:
@carolmariaabb if you have a look at the factoryImpl class, starting from the new method, there are a lot of references to private methods of this class, starting from _setDDMFormValuesAvailableLocales or _getDDMFormFieldParameterNames, which make use of a lot of them.

Reply:
Hi @crodriguezy,
What if you simplified the logic of `_hasDDMFormValuesEdited` so that you could move this method to `JournalEditArticleDisplayContext` without having to duplicate so much code?

I was discussing this with @rodrigopaulino and he suggested something like:

```
Map<String, String[]> parameterMap = httpServletRequest.getParameterMap();

for (Map.Entry<String, ?> entry : parameterMap.entrySet()) {
    if (StringUtil.contains(entry.getKey(), "ddm$$") &&
        StringUtil.contains(entry.getKey(), "_edited") &&
        GetterUtil.getBoolean(entry.getValue())) {

        return true;
    }
}
```

You could also some regex instead of doing `StringUtil.contains`.

What do you think?

Reply:
Hi @carolmariaabb , ok, I've just sent a new simplified PR to Echo team: https://github.com/liferay-echo/liferay-portal/pull/8321
I mentioned this one in case they have any objection, I hope they don't :)

Reply:
Thanks @crodriguezy. 
I'll close this PR for now, but feel free to reopen if you need to.